python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/dbt_packages/test_columns_metadata.py

```diff
@@ -403,7 +403,14 @@ def test_metadata_manifest_bigquery_fixture() -> Dict[str, Any]:
             marks=pytest.mark.snowflake,
             id="snowflake",
         ),
-        # TODO: test on bigquery
+        pytest.param(
+            "bigquery",
+            "test_metadata_manifest_bigquery",
+            # BigQuery does not support incremental_strategy='append'
+            ["count_star_implicit_alias_customers", "incremental_orders"],
```

Comment:
We end up passing it into the dbt invocation as the model name, IMO it's a bit easier to go this way rather than to convert the asset key to model name. We end up not having any key prefixes anyway, so it feels fairly equivalent + is less verbose.

Updated the code to pass excluded models to the CLI in all cases rather than using asset selection, which I think makes this a bit cleaner.